test/classes/report_test.rb

```diff
@@ -540,17 +538,17 @@ def test_project_tweaker_report
 
   private
 
-  def do_csv_test(report_type, obs, expect, &block)
+  def do_csv_test(report_type, obs, expect, user: nil, &block)
     query = Query.lookup(:Observation)
-    body = report_body(report_type, query)
+    body = report_body(report_type, query, user:)
     table = CSV.parse(body, col_sep: report_type.separator)
     assert_equal(query.num_results + 1, table.count)
     idx = query.results.sort_by(&block).index(obs)
     assert_equal(expect, table[idx + 1])
   end
 
-  def report_body(report_type, query)
-    report = report_type.new(query: query)
+  def report_body(report_type, query, user: nil)
+    report = report_type.new(query:, user:)
```

Comment:
Should this be the following? No reply necessary:
`report = report_type.new(query: query, user:)`

Reply:
I think i understand your question: only `:user` was a kwarg incoming, whereas `query` was an anonymous arg. But with the Ruby "kwarg" forwarding syntax, if the kwarg name matches a defined variable it will implicitly take that value if none given, even if the var was not originally a kwarg.